eo-maven-plugin/src/main/java/org/eolang/maven/AssembleMojo.java

```diff
@@ -203,6 +203,7 @@ public void exec() throws IOException {
             new Moja<>(ResolveMojo.class),
             new Moja<>(MarkMojo.class),
             new Moja<>(PlaceMojo.class),
+            new Moja<>(LatexMojo.class),
```

Comment:
@potatmen According to [this](https://github.com/objectionary/eo/issues/1206#issuecomment-1260503862), we don't need to add this Mojo to AssembleMojo. Check `GmiMojo` for more info